tests/greenheart/test_openmdao.py

```diff
@@ -8,14 +8,21 @@
 from hopp.simulation import HoppInterface
 from hopp.utilities import load_yaml
 
-from greenheart.tools.optimization.openmdao import HOPPComponent, TurbineDistanceComponent, BoundaryDistanceComponent
-
+from greenheart.tools.optimization.openmdao import GreenHeartComponent, HOPPComponent, TurbineDistanceComponent, BoundaryDistanceComponent
+from greenheart.tools.optimization.gc_PoseOptimization import PoseOptimization
+from greenheart.tools.optimization.gc_run_greenheart import run_greenheart
+from greenheart.simulation.greenheart_simulation import GreenHeartSimulationConfig
 import unittest
 
 solar_resource_file = Path(__file__).absolute().parent.parent.parent / "resource_files" / "solar" / "35.2018863_-101.945027_psmv3_60_2012.csv"
 wind_resource_file = Path(__file__).absolute().parent.parent.parent / "resource_files" / "wind" / "35.2018863_-101.945027_windtoolkit_2012_60min_80m_100m.srw"
-floris_input_file = Path(__file__).absolute().parent / "inputs" / "floris_input.yaml"
+floris_input_filename = Path(__file__).absolute().parent / "inputs" / "floris_input.yaml"
 hopp_config_filename = Path(__file__).absolute().parent / "inputs" / "hopp_config.yaml"
+hopp_config_steel_ammonia_filename = Path(__file__).absolute().parent / "test_hydrogen" / "input_files" / "plant" / "hopp_config.yaml"
+greenheart_config_onshore_filename = Path(__file__).absolute().parent / "test_hydrogen" / "input_files" / "plant" / "greenheart_config_onshore.yaml"
+turbine_config_filename = Path(__file__).absolute().parent / "test_hydrogen" / "input_files" / "turbines" / "osw_18MW.yaml"
+floris_input_filename_steel_ammonia = Path(__file__).absolute().parent / "test_hydrogen" / "input_files" / "floris" / "floris_input_osw_18MW.yaml"
+rtol = 1E-5
 
 class TestBoundaryDistanceComponent(unittest.TestCase):
```

Comment:
I think we should rewrite the `test_openmdeo` tests to follow the pytest structure we have established in HOPP and GreenHEART rather than introducing unittests.

Reply:
I agree, but I would like to put this in the next PR, which is pretty much ready